components/database/lmdb/manifests/sample-manifest.p5m

```diff
@@ -0,0 +1,9 @@
+file path=usr/include/lmdb.h
+file path=usr/lib/$(MACH64)/liblmdb.a
+file path=usr/lib/$(MACH64)/liblmdb.so
+file path=usr/lib/liblmdb.a
+file path=usr/lib/liblmdb.so
+file path=usr/share/man/man1/mdb_copy.1
+file path=usr/share/man/man1/mdb_dump.1
+file path=usr/share/man/man1/mdb_load.1
+file path=usr/share/man/man1/mdb_stat.1
```

Comment:
This file should be created by `gmake sample-manifest`. Then compare it with lmdb.p5m.

Reply:
I created it using "gmake sample-manifest MANIFEST_TEMPLATE=" as written in the wiki. Was that wrong? Is there some kind of better description?

Reply:
That sound to me like a outdated information. Pure `gmake sample-manifest` makes it as expected with the default template.